beets/ui/commands.py

```diff
@@ -1398,6 +1398,15 @@ def show_version(lib, opts, args):
 version_cmd.func = show_version
 default_commands.append(version_cmd)
 
+# database_location: return true if user
+# wants to create the parent directories.
+
+
+def database_dir_creation(path):
+    # Ask the user for a choice.
+    return ui.input_yn("{} does not exist, create it (Y/n)?"
```

Comment:
It might be helpful to more explicity describe the issue here; the path alone might not be enough to understand the problem: `The database directory {} does not exists, ...`?